web/client/configs/new.json

```diff
@@ -9,60 +9,123 @@
 			-20037508.34, -20037508.34,
 			20037508.34, 20037508.34
 		],
-		"layers": [
```

Comment:
same here, if you have enabled some formatter please disable it, at least for this project